web/templates/input_pill.hbs

```diff
@@ -20,8 +25,12 @@
             {{#if is_empty_string_topic}}
             {{sign}}topic: <span class="empty-topic-display">{{topic_display_name}}</span>
             {{else}}
+            {{#if is_email}}
```

Comment:
This can use `else if` instead of using another if-else block inside `else` block.